benchopt/tests/test_installcmd_solver.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+
+from benchopt.cli.main import run
+from benchopt.tests import SELECT_ONE_SIMULATED
+from benchopt.utils.temp_benchmark import temp_benchmark
+
+import tempfile
+import importlib.util
+from pathlib import Path
+
+
+def test_invalid_install_cmd():
+    # Solver with an invalid install command
+    invalid_solver = """
+    from benchopt import BaseSolver
+
+    class Solver(BaseSolver):
+        name = "invalid-solver"
+        install_cmd = "invalid_command"
+        requirements = []
+        def set_objective(self, X, y, lmbd): pass
+        def run(self, n_iter): pass
+        def get_result(self): return dict(beta=1)
+    """
+
+    with temp_benchmark(solvers=[invalid_solver]) as benchmark:
+        with pytest.raises(ValueError):
```

Comment:
To make sure you don't catch another error, it is good practice to check the error message:
```suggestion
        with pytest.raises(ValueError, match="invalid_command is not a valid"):
```